src/collection-browser.ts

```diff
@@ -541,26 +541,18 @@ export class CollectionBrowser
     });
   }
 
-  private selectedCreatorLetterChanged(): void {
-    this.creatorQuery = this.selectedCreatorFilter
-      ? `firstCreator:${this.selectedCreatorFilter}`
-      : undefined;
-  }
-
   private titleLetterSelected(
```

Comment:
JSDocs for newly added function would great.

Reply:
+1

Reply:
Added docs for the new getters and a few other methods that were missing them before. Thanks!